### ISSUE TITLE
Adding the APM integration feels a bit "bumpy"

### ISSUE THREAD

**Issue by philippkahr** [elastic/apm-server#7831]
Hi!

I am not entirely sure if this is for the apm-server repo or for the integrations.

So when adding the APM integration to a fleet server, either in ECK, or OnPrem you go through a few steps that aren't really necessary from a UX perspective.

## Current flow:
1. Go to Fleet
2. Select Agent Policy for fleet server
3. Click on `Add Integration`
4. Select APM in the top menu
5. Get routed to the APM Page (`/app/home#/tutorial/apm`)
6. Click again on `APM Integration`
7. Get routed to the Integrations page ( `/app/integrations/detail/apm-8.1.0/overview`)
8. Click on `add Elastic APM`
9. Scroll down to the bottom.
10. Reselect the `existing agent policy`
11. Select your agent policy.
12. Click `save and continue`

https://user-images.githubusercontent.com/12175559/162577732-f7fe5a2b-10b3-4626-b316-b967b24b18d7.mov

## Expected flow:

1. Go To Fleet
2. Select Agent Policy for fleet server
3. Clock on `Add Integration`
4. Select APM in the top menu
5. Get routed to the Integrations page ( `/app/integrations/detail/apm-8.1.0/overview`)
6. Click on `Add Elastic APM`
7. Have the existing agent policy preselected
8. Click on `save and continue`

Basically I don't want to get routed between multiple pages that forces me to loose the context and thus I have to reselect the agent policy, as well as performing a bunch of additional clicks that are confusing.

Another point to discuss would be if the `localhost:8200` and the `tls` disabled per default is the correct setting. If I just want to enable the APM server, I would then be confused why my app would not be able to reach my APM server. Maybe we should read the settings from fleet server, if it is deployed along side and set those as the same values?

**Comment by simitt** [comment]
cc @formgeist for visibility and providing an update here when the work on simplifying the onboarding flow is resumed.

**Comment by formgeist** [comment]
@philippkahr Would you mind if I convert this to a Kibana issue instead since it has to do with the UX of the integration flow and it'd be easier for me to gain visibility on the UI team for this? Not sure if any of the screenshots contain any secrets - otherwise I can anonymize them.

**Comment by philippkahr** [comment]
@formgeist No issues you can convert to a Kibana issue. Everything is from a demo system so no worries about secrets :)
